packages/core/src/components/link/test/default/link.e2e.ts

```diff
@@ -0,0 +1,53 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/link/test/default/index.html';
+
+test.describe('tds-link-default', () => {
+  test('is default link rendered correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    const tdsLink = page.getByTestId('tds-link-testid');
```

Comment:
Should we use `page.getByRole('link');` instead?

Reply:
I don't think so. That will target the slotted `<a>` no? Not the `<tds-link>`.

Reply:
@timrombergjakobsson  - I was thinking in first place of that, but tds-link component is not more than a wrapped that enforces styles on anchor tag that user is passing in, so I had to relay on testid.